apps/frontend/app/(client)/(code-editor)/_components/LeaderboardModalDialog.tsx

```diff
@@ -0,0 +1,98 @@
+import {
+  Dialog,
+  DialogContent,
+  DialogClose,
+  DialogTrigger,
+  DialogTitle
+} from '@/components/shadcn/dialog'
+import InfoIcon from '@/public/icons/info-icon.svg'
+import { X } from 'lucide-react'
+import Image from 'next/image'
+
+export function LeaderboardModalDialog() {
+  return (
+    <div className="flex">
+      <Dialog>
+        <DialogTrigger asChild>
+          <Image
+            className="cursor-pointer"
+            src={InfoIcon}
+            alt="info"
+            width={24}
+          />
+        </DialogTrigger>
+        <DialogContent
+          showDarkOverlay={true}
+          className="h-[450px] w-[700px] max-w-none gap-0 space-y-0 rounded-[16px] border-none bg-slate-900 pb-12 pl-[40px] pt-[40px] text-gray-300 [&>button:last-child]:hidden"
+        >
+          <DialogTitle className="sr-only">
+            Ranking System of Contest
+          </DialogTitle>
+          <DialogClose className="absolute left-[644px] top-[24px]">
+            <X className="h-[24px] w-[24px]" />
+          </DialogClose>
+          <div className="flex flex-col gap-[24px] overflow-x-auto">
```

Comment:
이런 식으로 바꿀 수 있는 px? 들은 바꾸면 더 좋을 것 같아요!
```suggestion
          <div className="flex flex-col gap-6 overflow-x-auto">
```